specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -1,15 +1,28 @@
 # ZK Fault Dispute Game
+<!-- All glossary references in this file. -->
+
+[g-zk-fault-proof]: ../../glossary.md#zk-fault-proof
+[g-l2-chain-derivation]: ../../glossary.md#l2-chain-derivation
+[g-sequencer-batch]: ../../glossary.md#sequencer-batch
+[g-dispute-game]: ../../experimental/zk-fault-dipute-game/overview.md
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [Overview](#overview)
-- [ZK Fault Dispute Game Creation](#zk-fault-dispute-game-creation)
-- [Challenge Creation](#challenge-creation)
-- [Dissection](#dissection)
-- [Proving Fault using ZKVM](#proving-fault-using-zkvm)
-- [Resolution of ZK Fault Dispute Game](#resolution-of-zk-fault-dispute-game)
+- [ZK Fault Dispute Game](#zk-fault-dispute-game)
```

Comment:
I'm not sure why these changes occurred. The changes were made by the `lint-specs-md-fix`, but should i revert them? @0xHansLee

Reply:
This may be due to the header I manually added, `# ZK Fault Dispute Game`, so please ignore this.
I'll make appropriate changes in the last touch.